lib/diagnostic.js

```diff
@@ -6,26 +6,32 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (string) {
+  let words = string.split(' ');
+  return words.replace(/\W+/,'').toUpperCase();
+
 };
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function (string) {
+  let words = string.split(' ');
+  return words.replace(/\W+/,'').toUpperCase();
 };
 
```

Comment:
How is this different than normalize words? Couldn't we just call the `normalizeWords` function?